anyway/secrets.py

```diff
@@ -12,7 +12,7 @@ def get(secret_name: str) -> str:
         with open(secrets_dir + secret_name) as secret_file:
             return secret_file.read()
     logging.info("No secret dir found.")
-    return os.environ[secret_name]
+    return os.environ.get(secret_name)
```

Comment:
Which callsites require this change? `password` alone does not justify avoiding the stricter `os.environ[]`, which gives more informative error message.

In case this change is necessary, please change the type signature to return `Optional[str]`:
```
from typing import Optional

def get(secret_name: str) -> Optional[str]:
    ...
```
and make sure that any call to `secrets.get()` handles `None` reasonably. Failing immediately with a clear exception is probably reasonable.

Reply:
Also, this change means `exists()` below may also be changed accordingly.

Reply:
@elazarg The thing is - not all secrets are available locally at this moment - hence locally it will fail

Reply:
However, this is an opportunity to rethink the above and perhaps create developer keys for all of these secrets

Reply:
But when it fails, it is a failure. Not something that should happen. There's no use for None here.